java/parser/src/test/java/org/sqlflow/parser/HiveQLParserAdaptorTest.java

```diff
@@ -0,0 +1,105 @@
+package org.sqlflow.parser;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import org.junit.Test;
+
+public class HiveQLParserAdaptorTest {
```

Comment:
These two tests have some code in common, maybe construct a base class to store the test case, and result testing functions and the sub-classes call the corresponding parser.

Reply:
Sure. I will refactorize it in the next PR.